rush.json

```diff
@@ -318,10 +318,10 @@
       "packageName": "@azure/abort-controller",
       "projectFolder": "sdk/core/abort-controller"
     },
-    {
-      "packageName": "@azure/core-http",
-      "projectFolder": "sdk/core/core-http"
-    },
+    // {
+    //   "packageName": "@azure/core-http",
```

Comment:
why?

Reply:
core-http doesn't build in the amqp-common branch anymore. It's a very old version (so old in fact that its version is still a carryover from ms-rest-js, so it's actually _newer_ than what we've released) and isn't needed for the purposes of building/testing amqp-common, so it's much easier to disable it than try to keep it up to date with what's in master over time.